testresult.txt

```diff
@@ -0,0 +1,14 @@
+Milyen fajlba mentsen: Fajl ahonnan betoltson: Nem sikerult megnyitni
```

Comment:
Ez egy oke teszteset, de kellene egy olyan is, hogy sikerul is beolvasnia egy fajlbol, meg kimentenie. Tehat fel van commitolva egy fajlszerkezetet leiro fajl, azt beolvassa, azon dolgozoik valamit, majd a vegen az eredmenyt kimenti egy masik fajlba. Es azt nezed meg, hogy annak a tartalma az-e, aminek lennie kell.

Reply:
@hegyhati done 😄